TAC/Meetings/2024/2024-05-16/CCC-TAC-Minutes-2024-05-16.md

```diff
@@ -0,0 +1,120 @@
+* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
+* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
+* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)
+
+
+#### Voting members of the TAC
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* AMD - David Kaplan / Harold Gilkey +
+* Arm - Nathaniel McCallum +   / Michael Lu
+* Google - Catherine Zhang +
+* Huawei - Zhipeng (Howard) Huang 
+* Intel - Dan Middleton (TAC chair) +  / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam / Henry Wang 
+* Microsoft - Alec Fernandez +
+* Nvidia - Fritz Alder +
+* Red Hat - Yash Mankad  / Ram Pai
+* TikTok - Mingshen Sun +  / Yao Zhang +
+
+   " + " indicates attending
+
+#### Other Attendees
+
+* Chris Ramming
+* Henk Birkholz
+* Hesham Elbakoury (Startup)
+* James Joshi (NSF)
+* Jeremy Epstein (EOP/OSTP)
+* Mark Novak (JP Morgan Chase)
+* Mic Bowman
+* Ofir Azoulay-Rozanes (Anjuna)
+* Wenhui Zhang (Tiktok)
+* Vini Jaiswal (TikTok)
+* Victor Lu
+
+* Helen Lau (LF)
+* Riaan Kleinhans (LF)
+* Sal Kimmich (LF)
+
+
+## Welcome
+
+Dan Middleton (DM) opened the call at 7:05 am and reviewed the antitrust policy. 
+
+## Agenda 
+
+DM reviewed the agenda for the meeting. 
+
+## Roll Call / Attendance
+
+Since no votes would be occurring in the meeting, attendance was skipped. 
+
+## Welcome New Community Members
+DM welcomed Helen back to the CCC meetings.
+
+* Mic Bowman introduced themselves.
+* Harold Gilkey introduced themselves as the alternative prep for AMD.
+* Chris Ramming reintroduced himself as the General Member Representative on the board.
+* James Joshi and Jeremy Epstein introduced themselves as guest speakers.
+
+### TAC Tech Talk PDaP
+
+* James Joshi (JJ) and Jeremy Epstein presented their work on Data Privacy. 
+* Chris Ramming (CR) mentioned working on a response to [Advancing Privacy Enhancing Technologies](https://www.nitrd.gov/rfi/2022/87-fr-35250/VMware-PET-RFI-Response-2022.pdf)
+* CR wanted clarification on the two ways to participate in this program. JJ asked the discussion be postponed till the end of the presentation.
+* DM asked if universities will be involved? JJ: Academia and small businesses will be involved.
+* Alec Fernandez asked how the academic sector is diverging from the commercial sector in PDaP? JJ continued to explain the relationship between academic research and the marketplace application of translational work. Discussion ensued.
+* JJ encouraged CCC partnership and participation in the funded collaboration in the project.
+* DM asked if there is a relationship with https://nsf-cdcc.org/?JJ confirmed he is aware of their work, but it is not directly related. 
```

Comment:
typo
>  https://nsf-cdcc.org/?JJ confirmed
 https://nsf-cdcc.org/ JJ confirmed

Reply:
Updated. Thank you